README.md

```diff
@@ -74,6 +74,23 @@ Airflow is not a streaming solution, but it is often used to process real-time d
 - **Elegant**:  Airflow pipelines are lean and explicit. Parameterizing your scripts is built into the core of Airflow using the powerful **Jinja** templating engine.
 - **Scalable**:  Airflow has a modular architecture and uses a message queue to orchestrate an arbitrary number of workers.
 
+## Version Life Cycle
+
+Apache Airflow version life cycle:
+
+| Version | Current Patch/Minor | State           | First Release | Limited Support | EOL/Terminated |
+|---------|---------------------|-----------------|---------------|-----------------|----------------|
+| 2       | 2.1.0               | Supported       | Dec 17, 2020  | Dec 2021        | TBD            |
+| 1.10    | 1.10.15             | Limited Support | Aug 27, 2018  | Dec 2020        | June 2021      |
```

Comment:
Should we be more specific than month/year when we can?

```suggestion
| 1.10    | 1.10.15             | Limited Support | Aug 27, 2018  | Dec 17, 2020    | June 17, 2021  |
```

Reply:
I like the part of making this more explicit too.

Reply:
@eladkal, missed the day in the EOL column, or was that intentional?